workspace/exercise_metadata.go

```diff
@@ -47,11 +47,11 @@ func NewSolution(dir string) (*Solution, error) {
 // Suffix is the serial numeric value appended to an exercise directory.
 // This is appended to avoid name conflicts, and does not indicate a particular
 // iteration.
-func (s *Solution) Suffix() string {
+func (s *ExerciseMetadata) Suffix() string {
```

Comment:
Likewise here (and throughout this file). I think in this case `em` is good, since it's in the context of a method receiver.